src/MvvmNano.Core/IPresenter.cs

```diff
@@ -10,6 +10,12 @@ namespace MvvmNano
     public interface IPresenter
     {
         /// <summary>
+        /// Changes the root page to the view of the given View model.
+        /// </summary>
+        /// <typeparam name="TViewModel"></typeparam>
+        void ChangeRootViewModel<TViewModel>() where TViewModel : MvvmNanoViewModel;
```

Comment:
Serious question: What's the purpose of this method? You basically want to navigate to another view model. That it's supposed to replace the root view is another story that shouldn't belong into the view model, but the presenter itself. And that's where `virtual Task OpenPageAsync(Page page)` inside the (custom) presenter comes into play.